assets/javascript/game.js

```diff
@@ -0,0 +1,260 @@
+}
+
+function createQuestionContainer(){
+   var questionDiv = document.createElement("div");
+    questionDiv.id = "questionContainer";
+    questionDiv.style.padding = "5px";
+    questionDiv.style.border = "thin black solid";
+    questionDiv.style.fontSize = "24";
+    questionDiv.style.margin = "auto";
+
+
+    document.getElementById('gamecontainer').appendChild(questionDiv);
+
+    questionDiv.innerHTML = "Question: " + questions[currentQuestion];
+    
+}
+
+function createAnswerContainer(){
+    var answerDiv = document.createElement("div");
+    answerDiv.id = "answerContainer";
+
+    var aWidth = answers[currentQuestion].length*62;
+    answerDiv.style.width = aWidth + "px";
+    answerDiv.style.height = "75px";
+    answerDiv.style.margin = "auto";
+    answerDiv.style.marginTop = "20px";
+
+    document.getElementById('gamecontainer').appendChild(answerDiv);
+    
+    currentLetter = 0;
+
+    answers[currentQuestion].split('').forEach(function(curLetter){
+        var letterDiv = document.createElement("div");
+        letterDiv.id = "letter" + currentLetter;
+        letterDiv.style.width = "60px";
+        letterDiv.style.height = "100px";
+
+        if(isLetter(curLetter)){
+            letterDiv.style.border = "thin black solid";
+        }
+        letterDiv.style.cssFloat = "left";
+        letterDiv.style.textAlign = "center";
+        
+        document.getElementById('answerContainer').appendChild(letterDiv);
+
+        currentLetter++;
+    });
+
+    
+    }
+function createBillboardContainer() {
+
+    var billboardDiv = document.createElement("div");
+    billboardDiv.id = "billboardContainer";
+    billboardDiv.style.width = "400px";
+    billboardDiv.style.height = "100px";
+    billboardDiv.style.border = "thin black solid";
+    billboardDiv.style.visibility = "hidden";
+    billboardDiv.style.position = "fixed";
+    billboardDiv.style.left = "420px";
+    billboardDiv.style.top = "250px";
+    billboardDiv.style.marginleft = "-200px";
+    billboardDiv.style.marginRight = "34px";
+    billboardDiv.style.paddingTop = "24px";
+    billboardDiv.style.textAlign = "center";
+    billboardDiv.style.background = "crimson";
+    billboardDiv.style.fontsize = "24px";
+    billboardDiv.style.color = "whitesmoke";
+    
+
+    document.getElementById("gamecontainer").appendChild(billboardDiv);
+
+
+}
+function updateBillboardContainer(theText) {
+    var extraText = "<br><br>Hit ENTER to continue";
+    if(endgame == true){
+        extraText = "<br><br>Hit ENTER to restart game. ";
+
+ }
+ document.getElementById('billboardContainer').innerHTML = theText + extraText;
+
+ document.removeEventListener("keydown", gameTurnLoop);
+
+ document.addEventListener("keydown", billboardListener, false);
+
+ document.getElementById('billboardContainer').style.visibility = "visible";
+
+}
+
+function billboardListener(e){
+    var keyCode = e.keyCode;
+
+    if(keyCode==13) {
```

Comment:
with codes like this to make it more readable you can create a variable at the top of the page
```javascript
var ENTER_KEY = 13;
```
and then down here at this line you can make it 
```javascript
if (keyCode === ENTER_KEY) {
```